content/travel bags.md

```diff
@@ -0,0 +1,25 @@
+---
+title: Travel Bags
+menu: "main"
+type: "product"
+weight: 8
+images:
+ - "/img/travelDITTYBAG.jpg"
```

Comment:
I think you'll need to add this file also.  I don't see it in the master branch currently.